src/encoded/static/components/item-pages/components/HiGlass/HiGlassContainer.js

```diff
@@ -917,29 +922,34 @@ export class HiGlassContainer extends React.PureComponent {
      *
      * - this.updateCurrentDomainsInStorage is bound to 'location' change event.
      * - TODO: onDrag/Drop stuff.
+     *
+     * @deprecated
      */
-    bindHiGlassEventHandlers(){
+    bindHiGlassEventHandlers(attemptCount = 0){
```

Comment:
This code is only used for FileView, ExpSetView, & ExpView for the _auto-generated_ HiGlassViewConfigs.
It _will_ be removed. Currently this code binds a handler which listens to location changes and then updates location per filetype in browser localStorage to allow user to browse same location of different files.

Further work on this requires UX feedback/planning.